docs/requirements.txt

```diff
@@ -3,7 +3,7 @@
 -r ../requirements.txt
 alabaster_jupyterhub
 autodoc-traits
-https://github.com/pandas-dev/pandas-sphinx-theme/archive/ade576c92cfe46a372b6783fb83f45c44fc67976.zip
+pydata-sphinx-theme
```

Comment:
There's one extra line that needs to be changed :rocket: : :
https://github.com/jupyterhub/jupyterhub/blob/4e8acc71c671d39b0ae6f140d894c983bc448233/docs/source/conf.py#L77

Reply:
good catch!